conf/layer.conf

```diff
@@ -8,4 +8,4 @@ BBFILE_COLLECTIONS += "meta-rpb"
 BBFILE_PATTERN_meta-rpb := "^${LAYERDIR}/"
 BBFILE_PRIORITY_meta-rpb = "8"
 
-LAYERSERIES_COMPAT_meta-rpb = "thud"
+LAYERSERIES_COMPAT_meta-rpb = "thud warrior"
```

Comment:
should it be just "warrior" instead? and we should warrior branches..

Reply:
no, master can be compatible with multiple branches. The changes is accurate, our meta-rpb master branch is currently compatible with both thud and warrior.